Tests/HederaE2ETests/Contract/ContractIdPopulation.swift

```diff
@@ -0,0 +1,59 @@
+/*
+ * ‌
+ * Hedera Swift SDK
+ * ​
+ * Copyright (C) 2022 - 2024 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import Hedera
+import XCTest
+
+internal final class ContractIdPopulation: XCTestCase {
```

Comment:
needs negative tests (IE, succeeds when something goes wrong and that something is the thing that's supposed to go wrong, we have others like that)

Reply:
Doesn't really serve any real purpose in the overall functionality. Omitting.